ppa/archive/eebo_tcp.py

```diff
@@ -27,9 +27,8 @@ class Page(xmlmap.XmlObject):
     # page beginning tags delimit content instead of containing it;
     # use following axis to find all text nodes following this page beginning
     text_contents = xmlmap.StringListField("following::text()")
-
-    # notes on or after this page, for managing footnote text
-    notes = xmlmap.NodeListField("following::NOTE", xmlmap.XmlObject)
+    # count following notes as a quick check to bail out of note detection logic
+    has_notes = xmlmap.IntegerField("count(following::NOTE)")
```

Comment:
Since this is only used for boolean tests, make it a boolean. Otherwise, consider renaming it to reflect that it isn't a boolean.

Reply:
eulxml has a boolean field but it was written for reading and writing boolean values, but I couldn't get it to work here - I think it must assume string values and can't handle boolean or integer values; I'll add a comment

Reply:
Sorry, I keep forgetting this is all eulxml